client/src/main/com/sinch/sdk/domains/voice/adapters/converters/CalloutsDtoConverter.java

```diff
@@ -0,0 +1,128 @@
+import com.sinch.sdk.domains.voice.models.dto.v1.CustomCalloutRequestDto;
+import com.sinch.sdk.domains.voice.models.dto.v1.DestinationDto;
+import com.sinch.sdk.domains.voice.models.dto.v1.DestinationDto.TypeEnum;
+import com.sinch.sdk.domains.voice.models.dto.v1.GetCalloutResponseObjDto;
+import com.sinch.sdk.domains.voice.models.dto.v1.TtsCalloutRequestDto;
+import com.sinch.sdk.domains.voice.models.requests.CalloutRequestParameters;
+import com.sinch.sdk.domains.voice.models.requests.CalloutRequestParametersConference;
+import com.sinch.sdk.domains.voice.models.requests.CalloutRequestParametersCustom;
+import com.sinch.sdk.domains.voice.models.requests.CalloutRequestParametersTTS;
+import com.sinch.sdk.domains.voice.models.requests.ConferenceDtfmOptions;
+import com.sinch.sdk.domains.voice.models.requests.Destination;
+import com.sinch.sdk.domains.voice.models.requests.DestinationNumber;
+import com.sinch.sdk.domains.voice.models.requests.DestinationUser;
+
+public class CalloutsDtoConverter {
+
+  public static CalloutRequestDto convert(CalloutRequestParameters client) {
+
+    if (client instanceof CalloutRequestParametersConference) {
+      CalloutRequestParametersConference parameters = (CalloutRequestParametersConference) client;
+      return convert(parameters);
+    } else if (client instanceof CalloutRequestParametersTTS) {
+      CalloutRequestParametersTTS parameters = (CalloutRequestParametersTTS) client;
+      return convert(parameters);
+    } else if (client instanceof CalloutRequestParametersCustom) {
+      CalloutRequestParametersCustom parameters = (CalloutRequestParametersCustom) client;
+      return convert(parameters);
+    }
+    return null;
+  }
+
+  public static String convert(GetCalloutResponseObjDto dto) {
+    if (null == dto) {
+      return null;
+    }
+    return dto.getCallId();
+  }
+
+  private static CalloutRequestDto convert(CalloutRequestParametersConference client) {
+
+    ConferenceCalloutRequestDto dto =
+        new ConferenceCalloutRequestDto().destination(convert(client.getDestination()));
+
+    client.getCli().ifPresent(f -> dto.setCli(f.stringValue()));
+    client.getDtfm().ifPresent(dto::setDtmf);
+    client.getCustom().ifPresent(dto::setCustom);
+
+    dto.conferenceId(client.getConferenceId());
+    client.getDtfmOptions().ifPresent(f -> dto.setConferenceDtmfOptions(convert(f)));
+
+    client.getMaxDuration().ifPresent(dto::setMaxDuration);
+    client.getEnableAce().ifPresent(dto::setEnableAce);
+    client.getEnableDice().ifPresent(dto::setEnableDice);
+    client.getEnablePie().ifPresent(dto::setEnablePie);
+    client.getLocale().ifPresent(dto::setLocale);
+    client.getGreeting().ifPresent(dto::setGreeting);
+    client.getMohClass().ifPresent(f -> dto.setMohClass(f.value()));
+    client.getDomain().ifPresent(f -> dto.setDomain(f.value()));
+
+    return new CalloutRequestDto()
+        .method(MethodEnum.CONFERENCECALLOUT.getValue())
```

Comment:
Is this enum name generated from the spec? Otherwise, you may rename it `CONFERENCE_CALLOUT`

Reply:
It come from generated file from OAS: https://github.com/sinch/sinch-sdk-java/blob/feat/voice-callout/openapi-contracts/src/main/com/sinch/sdk/domains/voice/models/dto/v1/CalloutRequestDto.java#L37-L43

Reply:
/me crying silently

Reply:
OAS file: 
```
          enum:
            - conferenceCallout
            - ttsCallout
            - customCallout
```
So let say the generator just do the job...